docs/user_guide/advanced.rst

```diff
@@ -18,7 +18,7 @@ First, as a fourth column in the parameter file:
     P3 0.0 5.0 Group_2
     ...etc.
 
-Or in the `problem` dictionary:
+Second, as an additional key in the :code:`problem` dictionary:
```

Comment:
"Second" implies something the user is expected to do, whereas this is providing an alternate approach.

```suggestion
Or, as an additional key in the :code:`problem` dictionary:
```